torchinfo/layer_info.py

```diff
@@ -12,6 +12,17 @@
 ]
 
 
+def rgetattr(obj: torch.nn.Module, attr: str) -> torch.Tensor:
+    """Get the tensor submodule called attr from obj."""
+    for i in attr.split("."):
```

Comment:
Let's use a dfferent variable since `i` is typically reserved for numbers

Reply:
Would `attr_i` be fine (to say some part of the variable `attr`)?